src/pages/settings/Wallet/WalletPage/WalletPage.js

```diff
@@ -66,6 +68,8 @@ function WalletPage({bankAccountList, betas, cardList, fundList, isLoadingPaymen
     const hasAssignedCard = !_.isEmpty(cardList);
     const shouldShowEmptyState = !hasBankAccount && !hasWallet && !hasAssignedCard;
 
+    const {isPendingOnfidoResult} = userWallet;
```

Comment:
Just confirming, if `isPendingOnfidoResult` doesn't exist in the userWallet, then it will be undefined so do we prefer lodashGet in these cases?